changelogs/changelog.yaml

```diff
@@ -23,3 +23,142 @@ releases:
+      - add no_log to prevent credentials leak (https://github.com/redhat-cop/cloud.aws_ops/pull/92).
+      - cloud.aws_ops.aws_manage_cloudtrail_encryption - fix condition logic to match
+        expected Cloudtrail events and add extra_vars to pass rulebook variables to
+        playbooks called in actions (https://github.com/redhat-cop/cloud.aws_ops/pull/86).
+      - cloud.aws_ops.aws_restore_cloudtrail - provide `key_prefix` default so it
+        doesn't error if not present (https://github.com/redhat-cop/cloud.aws_ops/pull/86).
+      - cloud.aws_ops.aws_restore_kms_key - fix conditional value to properly retrieve
+        KMS key ARN from ansible-rulebook event variable (https://github.com/redhat-cop/cloud.aws_ops/pull/86).
+      - cloud.aws_ops.enable_cloudtrail_encryption_with_kms - fix incorrect fact name
+        for retrieved trail info and provide `s3_key_prefix` default so it doesn't
+        error if not present (https://github.com/redhat-cop/cloud.aws_ops/pull/86).
+      - cloud.aws_ops.webapp - update RDS engine from deprecated version (https://github.com/redhat-cop/cloud.aws_ops/pull/86).
+      - cloud.aws_ops.webapp - update webapp create task to use provided variables
+        instead of hard-coding values in some places (https://github.com/redhat-cop/cloud.aws_ops/pull/86).
+      - fix and update integration tests target test_manage_vpc_peering (https://github.com/redhat-cop/cloud.aws_ops/pull/61).
+      minor_changes:
+      - Add a playbook to deploy a simple flask web app into high availability architecture
+        (https://github.com/redhat-cop/cloud.aws_ops/pull/97).
+      - Convert deploy_flask_app playbook to role (https://github.com/redhat-cop/cloud.aws_ops/pull/85).
+      - awsconfig_apigateway_with_lambda_integration - new role to create API gateway
+        with Lambda integration
+      - cloud.aws_ops.upload_file_to_s3 - A playbook to upload file from local filesystem
+        into S3 bucket (https://github.com/redhat-cop/cloud.aws_ops/pull/88).
+      - create_rds_global_cluster - new role to create aurora global cluster with
+        a primary and a replica cluster in different regions.
+    fragments:
+    - 20230426-update_readme_and_runtime.yml
+    - 20230804-backup-role-bugfixes.yaml
+    - 20230817-update_for_controller.yml
+    - 20230821-upload_file_to_s3.yaml
+    - 20230822-docs_and_bugfixes.yml
+    - 20230823-update-ci-sanity-test-matrix.yaml
+    - add_deploy_flask_app_role.yaml
+    - ansible_lint_6_19.yml
+    - ansible_lint_updates.yml
+    - automation_hub_update_readme.yaml
+    - aws_setup_credentials_add_no_log.yml
+    - awsconfig_apigateway.yml
+    - create_rds_global_cluster_role.yml
+    - customized_ami_doc_update.yaml
+    - fix_manage_vpc_peering_integration_test.yml
+    - fix_sanity_issues.yaml
+    - transit_gateway_doc_update.yaml
+    - update_ansible_lint_action.yml
+    - webapp_in_HA.yaml
+    release_date: '2023-11-15'
```

Comment:
We might have to change this based on when we release. I hope that will be take care of.